dspace-spring-rest/pom.xml

```diff
@@ -169,6 +169,20 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 			<version>${spring-boot.version}</version>
+                        <exclusions>
```

Comment:
Could we add a simple comment here that says something like:

<!-- Temporary exclusions until DS-3135 is fixed. Once we are using log4j v2, we should update Spring Boot to utilize it per these instructions: https://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-configure-log4j-for-logging -->